core/base/discreteGradient/DiscreteGradient.cpp

```diff
@@ -233,3 +239,24 @@ int DiscreteGradient::setManifoldSize(
 
   return 0;
 }
+
+#ifdef TTK_ENABLE_MPI
+void DiscreteGradient::setCellToGhost(const int cellDim,
+                                      const SimplexId cellId) {
+  if(cellDim == 0) {
+    (*gradient_)[0][cellId] = GHOST_GRADIENT;
+  }
+
+  if(cellDim == 1) {
+    (*gradient_)[1][cellId] = GHOST_GRADIENT;
```

Comment:
If I remember correctly, you might also want to put GHOST_GRADIENT in (*gradient_)[2] for edges and (*gradient_)[4] for triangles (each pair of gradient_ vectors holds mapping between facets and co-facets of increasing dimension).